lib/Models/SdmxJson/SdmxJsonStructureMessage.ts

```diff
@@ -116,62 +147,9 @@ export type DataStructure = MaintainableType & {
-        /**
-         * The position attribute specifies the position of the dimension in the data structure definition, starting at 0. It is optional as the position of the dimension in the key descriptor (DimensionList element) always takes precedence over the value supplied here. This is strictly for informational purposes only.
-         */
-        position?: number;
-        /**
-         * The type attribute identifies whether then dimension is a measure dimension, the time dimension, or a regular dimension. Although these are all apparent by the element names, this attribute allows for each dimension to be processed independent of its element as well as maintaining the restriction of only one measure and time dimension while still allowing dimension to occur in any order.
-         */
-        type?: "Dimension" | "MeasureDimension" | "TimeDimension";
-        /**
-         * Urn reference to a concept where the identification of the concept scheme which defines it is contained in another context.
-         */
-        conceptIdentity?: string;
-        /**
-         * ConceptRoles references concepts which define roles which this dimension serves. If the concept from which the attribute takes its identity also defines a role the concept serves, then the isConceptRole indicator can be set to true on the concept identity rather than repeating the reference here.
-         */
-        conceptRoles?: string[];
-        localRepresentation?: MeasureDimensionRepresentationType;
-        [k: string]: unknown;
-      })[];
-      timeDimensions?: (IdentifiableTypeWithNCNameID & {
-        /**
-         * The position attribute specifies the position of the dimension in the data structure definition, starting at 0. It is optional as the position of the dimension in the key descriptor (DimensionList element) always takes precedence over the value supplied here. This is strictly for informational purposes only.
-         */
-        position?: number;
-        /**
-         * The type attribute identifies whether the dimension is a measure dimension, the time dimension, or a regular dimension. Although these are all apparent by the element names, this attribute allows for each dimension to be processed independent of its element as well as maintaining the restriction of only one measure and time dimension while still allowing dimension to occur in any order.
-         */
-        type?: "Dimension" | "MeasureDimension" | "TimeDimension";
-        /**
-         * Urn reference to a concept where the identification of the concept scheme which defines it is contained in another context.
-         */
-        conceptIdentity?: string;
-        localRepresentation?: TimeDimensionRepresentationType;
-        [k: string]: unknown;
-      })[];
+      dimensions?: Dimension[];
+      measureDimensions?: Dimension[];
+      timeDimensions?: Dimension[];
       [k: string]: unknown;
     };
     groups?: (IdentifiableType & {
```

Comment:
Here I have just moved a few chunks into their own types (`Dimension` and `Attribute`)